skimage/transform/__init__.py

```diff
@@ -12,6 +12,7 @@
 from ._warps import swirl, resize, rotate, rescale, downscale_local_mean
 from .pyramids import (pyramid_reduce, pyramid_expand,
                        pyramid_gaussian, pyramid_laplacian)
+from seam_carving import seam_carve
```

Comment:
I think making this an absolute import will fix most of the Travis errors. 
`from .seam_carving import seam_carve`